changelog.md

```diff
@@ -25,6 +25,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add file explorer on text view
 - Add plugin manager
 - Display current git branch of project in text view
+- Add menu to list branches
```

Comment:
1. should last line
2. `Add menu to list git repository branches`

Reply:
Don't care of order, because before first release I will make a pre-release and rework on changelog :)
Done for the better sentence